@kiva/kv-components/vue/stories/KvEditButton.stories.js

```diff
@@ -0,0 +1,64 @@
+import KvEditButton from '../KvEditButton.vue';
+import KvButton from '../KvButton.vue';
+
+export default {
+	title: 'KvEditButton',
+	component: KvEditButton,
+	argTypes: {
+		visible: {
+			control: 'boolean',
+			defaultValue: false,
+		},
+		title: {
+			control: 'text',
+			defaultValue: 'Edit',
+		},
+		variant: {
+			control: 'select',
+			options: ['lightbox', 'alert'],
+			defaultValue: 'lightbox',
+		},
+		preventClose: {
+			control: 'boolean',
+			defaultValue: false,
+		},
+	},
+	args: {
+		visible: false,
+		title: '',
+		variant: 'lightbox',
+		preventClose: false,
+	},
+};
+
+const DefaultTemplate = (args, { argTypes }) => ({
+	props: Object.keys(argTypes),
+	components: { KvEditButton, KvButton },
+	template: `
+		<div>
+			<p>The lightbox is visible: {{isLightboxVisible}}</p>
```

Comment:
This value doesn't currently change when the lightbox is opened. Maybe it isn't supposed to?